sw/airborne/subsystems/ahrs/ahrs_int_cmpl_quat.c

```diff
@@ -270,7 +270,11 @@ void ahrs_icq_update_accel(struct Int32Vect3 *accel, float dt)
 #define ACC_FROM_CROSS_FRAC INT32_RATE_FRAC + INT32_SPEED_FRAC - INT32_ACCEL_FRAC - COMPUTATION_FRAC
 
     const struct Int32Vect3 vel_tangential_body =
+#ifdef VERTICAL_WING_HYBRID
+      {0, 0, -(ahrs_icq.ltp_vel_norm >> COMPUTATION_FRAC)};
```

Comment:
I made the change to use vel on all axes a long time ago for @EwoudSmeur , not sure if it has been tested though:
https://github.com/flixr/paparazzi/commit/1c05c0a880bc33612707703fe8b22c5b10349cfc